src/tests/test_rest_framework_dso/test_serializers.py

```diff
@@ -137,7 +137,18 @@ def test_pagination_many(drf_request, movie):
 @pytest.mark.django_db
 def test_fields_limit_works(api_rf, movie):
     """Prove that serializer can limit output fields."""
-    django_request = api_rf.get("/", {"fields": "name"})
+    _test_fields_limit(api_rf, movie, "name")
+
+
+@pytest.mark.django_db
+def test_fields_limit_negative(api_rf, movie):
+    """Prove that the serializer can omit output fields on request."""
+    # Omit everything but the name.
+    _test_fields_limit(api_rf, movie, "-category_id")
+
+
```

Comment:
Use `@pytest.mark.parametrize("fields", ["name", "-category_id"])` for this.

Reply:
Done.